misc/assets/inforequests.svg

```diff
@@ -0,0 +1,248 @@
+<svg version="1.1" viewBox="0 0 1000 1350" xmlns="http://www.w3.org/2000/svg">
+  <style>
+      text {
+        font-family: sans-serif;
+      }
+      .external{
+        stroke-dasharray: 3pt;
+      }
+      .caption {
+        font-size: 12pt;
+        text-anchor: middle;
+        dominant-baseline: central;
+      }
+      .note {
+        font-size: 9pt;
+        text-anchor: middle;
+        dominant-baseline: ideographic;
+      }
+      .left-rotation{
+        transform: rotate(-90deg);
+      }
+      .right-rotation{
+        transform: rotate(90deg);
+      }
+      .relation {
+        fill: none;
+        stroke: black;
+        stroke-width: 1pt;
+      }
+      .relation.from-many {
+        marker-start: url(#relation-many)
+      }
+      .relation.to-many {
+        marker-end: url(#relation-many)
+      }
+      .relation.from-optional {
+        marker-start: url(#relation-optional)
+      }
+      .relation.to-optional {
+        marker-end: url(#relation-optional)
+      }
+      .relation.to-link_attribute {
+        marker-end: url(#link_attribute)
+      }
+      .dependency {
+        fill: none;
+        stroke: black;
+        stroke-width: 1pt;
+        marker-end: url(#dependency);
+      }
+  </style>
+
+  <defs>
+    <marker id="relation-many" orient="auto-start-reverse" markerWidth="6" markerHeight="6" refX="6" refY="3">
+      <circle cx="3" cy="3" r="3" />
+    </marker>
+    <marker id="relation-optional" orient="auto-start-reverse" markerWidth="6" markerHeight="6" refX="6" refY="3">
+      <circle cx="3" cy="3" r="2.5" fill="white" stroke="black" stroke-width="1" />
+    </marker>
+    <marker id="link_attribute" orient="-90" markerWidth="10" markerHeight="6" refX="5" refY="0">
+      <path d="M 1 0.5 A 1.5 2, 0, 0 0, 8.5 0.5" fill="white" stroke="black" />
+    </marker>
+    <marker id="dependency" markerWidth="6" markerHeight="6" refX="3" refY="0">
+      <polygon points="0,6 6,6 3,0" />
+    </marker>
+    <g id="model">
+      <rect class="box" x="-100" y="-20" width="200" height="40" fill="none" stroke="black" stroke-width="1pt" />
+    </g>
+    <g id="model-narrow">
+      <rect class="box" x="-50" y="-20" width="100" height="40" fill="none" stroke="black" stroke-width="1pt" />
+    </g>
+  </defs>
+
+  <g transform="translate(100 100)">
+    <use href="#model-narrow" class="external"/>
+    <text class="caption">User</text>
+    <path class="relation from-one to-many" d="M 0,20 0,80" />
+    <path class="relation from-one to-many" d="M 50,0 100,0" />
```

Comment:
Ako som pisal minule, relacia ma vychadzat z modelu, v ktorom je definovana. Ako hint mozes brat to, ze ak je relacia definovana v externom modeli, tak je tiez externa, nie?

Pls oprav vsetky relation ciary, ktore su naopak.

Reply:
:ok: